src/main/java/io/vertx/core/http/impl/HttpChannelConnector.java

```diff
@@ -116,6 +117,13 @@ public void deactivate(HttpClientConnection conn) {
     }
   }
 
+  private boolean isExcluded(ProxyOptions options, String targetProxy) {
```

Comment:
Remove this as it's no longer needed; see comment for `HttpClientImpl.useProxy()`.